Create zh_tw.lang

aa89227-zz opened the pull request.

aa89227-zz committed:
Create zh_tw.lang

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/thexaero/Better-PVP-Translations?pullRequest=5) <br/>All committers have signed the CLA.

thexaero commented:
Thank you very much!
Sorry for not responding sooner. I'll try to review the translation soon and provide feedback.

thexaero commented on zh_tw.lang:
```diff
@@ -0,0 +1,209 @@
+gui.xaero_confirm =確認
```
Ok, there definitely isn't supposed to a space before the "=" on every line. Please fix that.

thexaero commented on zh_tw.lang:
```diff
@@ -0,0 +1,209 @@
+gui.xaero_OK =確定
+gui.xaero_add_edit =添加/編輯
+gui.xaero_new_waypoint =新的路徑點
+gui.xaero_initial =初始化
```
Isn't "初始" more correct?

the-eric-kwok replied:
Due to the lack of full context, it can't be determined to use "初始" or "初始化" or even "首字母", but it seems to be a noun? If so, then "首字母" is the correct one.

thexaero commented on zh_tw.lang:
```diff
@@ -0,0 +1,209 @@
+gui.xaero_initial =初始化
+gui.xaero_edit_waypoint =編輯路徑點
+gui.xaero_armour_status_settings =盔甲狀態設置
+gui.xaero_choose_a_preset =預設方案
```
Is this correct? It translates to "preset plan" with Google Translate, which isn't correct.

the-eric-kwok replied:
This is correct in Chinese statement. Which means to select from a bunch of presets.

thexaero commented on zh_tw.lang:
```diff
@@ -0,0 +1,209 @@
+gui.xaero_edit_waypoint =編輯路徑點
+gui.xaero_armour_status_settings =盔甲狀態設置
+gui.xaero_choose_a_preset =預設方案
+gui.xaero_choose_an_interface =界面
```
Is this correct for "Choose an interface"? Translates as "interface" for me.

the-eric-kwok replied:
I think the correct translation should be "選擇一個界面".

thexaero commented on zh_tw.lang:
```diff
@@ -0,0 +1,209 @@
+gui.xaero_not_ingame =你必須先進入遊戲開啟編輯模式
+gui.xaero_welcome_edit_mode =歡迎使用編輯模式
+gui.xaero_howto_select =左鍵點擊選擇界面
+gui.xaero_howto_drag =左鍵單擊並初始化界面
```
Is this correct? Translates as "initialize" instead of "drag" for me.

the-eric-kwok replied:
It should be "左鍵單擊並拖動來移動界面".

thexaero commented on zh_tw.lang:
```diff
@@ -0,0 +1,209 @@
+gui.xaero_welcome_edit_mode =歡迎使用編輯模式
+gui.xaero_howto_select =左鍵點擊選擇界面
+gui.xaero_howto_drag =左鍵單擊並初始化界面
+gui.xaero_howto_deselect =快捷取消選擇界面
```
Doesn't seem like an instruction about how to deselect an interface at all to me. Might be wrong.

the-eric-kwok replied:
It should be "右鍵單擊來取消選擇界面"

thexaero commented on zh_tw.lang:
```diff
@@ -0,0 +1,209 @@
+gui.xaero_howto_drag =左鍵單擊並初始化界面
+gui.xaero_howto_deselect =快捷取消選擇界面
+gui.xaero_howto_center = C鍵固定界面居中
+gui.xaero_different_centered =某些界面有著不同的居中方式
```
Seems incorrect when translated back. Does this mean "some interfaces look differently centered" and not "have different centering methods"?

the-eric-kwok replied:
The word ”方式“ sometimes means "style" in Chinese statement, but I prefer using "樣式"

thexaero commented on zh_tw.lang:
```diff
@@ -0,0 +1,209 @@
+gui.xaero_howto_deselect =快捷取消選擇界面
+gui.xaero_howto_center = C鍵固定界面居中
+gui.xaero_different_centered =某些界面有著不同的居中方式
+gui.xaero_howto_flip = F鍵插入界面
```
Is this correct for "flipping" interfaces? Translates back as "insert".

the-eric-kwok replied:
No, "flipping" should be "翻轉" in Chinese.

thexaero commented on zh_tw.lang:
```diff
@@ -0,0 +1,209 @@
+gui.xaero_better_pvp_settings =更好的PvP設置
+gui.xaero_server_disabled =服務器可能禁止了某些Mod特性
+gui.xaero_waypoints =路徑點
+gui.xaero_box_potion_effects =藥水效果界面
+gui.xaero_box_armour_status =裝備狀態界面
+gui.xaero_box_sprint =一鍵衝鋒
+gui.xaero_box_sneak =潛行開關
+gui.xaero_box_notifications =屏幕提醒，某些低溫/低血量/爆炸/弓箭。
+gui.xaero_box_xp =經驗顯示
+gui.xaero_box_quick_use =快速使用動作條物品
+gui.xaero_box_archery =弓箭數量顯示
+gui.xaero_box_minimap =小地圖顯示
```
All of these are supposed to **explain** the features, not just name them.

thexaero commented on zh_tw.lang:
```diff
@@ -0,0 +1,209 @@
+gui.xaero_flipped =翻轉
+gui.xaero_press_f = F鍵
+gui.xaero_preset_fancy =幻想
+gui.xaero_preset_classic =普通
```
Translates back as "normal" to me, which isn't correct. The classic preset isn't "normal".

thexaero commented on zh_tw.lang:
```diff
@@ -0,0 +1,209 @@
+gui.xaero_fbo =測試版
+gui.xaero_potion_status_settings =藥水狀態設置
+gui.xaero_potion_names =藥水效果名稱
+gui.xaero_opacity =不更長
```
Translates back as "no longer" for me, which is weird.

the-eric-kwok replied:
Weird translation.. Should be "不透明度"

thexaero commented on zh_tw.lang:
```diff
@@ -0,0 +1,209 @@
+gui.xaero_waypoints_scale =標記大小
+gui.xaero_block_colours =方塊顏色
+gui.xaero_accurate =準確
+gui.xaero_vanilla =普通
```
The vanilla is probably the least "normal" of the two options, but this translates back as "normal" for me.

the-eric-kwok replied:
I preferred "原版" as it represents the original Minecraft style.

thexaero commented on zh_tw.lang:
```diff
@@ -0,0 +1,209 @@
+gui.xaero_waypoint_set =設置
+gui.xaero_delete_set =刪除設置
+gui.xaero_delete_set_message =你確定你想刪除當前設置嗎
+gui.xaero_delete_set_message2 =此設置重置當前世界中可移除
```
Please translate this back for me. Google gives me something weird.

the-eric-kwok replied:
Does the word "set" meaning of "a set of waypoints"? If so, then it should be "集合", not "設置". 
And for the weird line, I think it should be "此集合將從此世界中刪除！"

thexaero commented on zh_tw.lang:
```diff
@@ -0,0 +1,209 @@
+gui.xaero_delete_set_message =你確定你想刪除當前設置嗎
+gui.xaero_delete_set_message2 =此設置重置當前世界中可移除
+gui.xaero_toggle_waypoints =切換路徑點
+gui.xaero_dots_scale =實體顯示範圍
```
This should be specific to "dots". Is it?

the-eric-kwok replied:
should be "表示實體的點的大小"

thexaero commented on zh_tw.lang:
```diff
@@ -0,0 +1,209 @@
+gui.xaero_entity_info =實體信息
+gui.xaero_toggle_slime =切換史萊姆區塊鏈
+gui.xaero_toggle_grid =切換區塊網格
+gui.xaero_compass_over_wp =指南針上的路徑點
```
This translates back as "waypoints over compass" for me. Which is the opposite of what it should mean. Is this correct?

the-eric-kwok replied:
It should be "路徑點顯示與指南針之上"

thexaero commented on zh_tw.lang:
```diff
@@ -0,0 +1,209 @@
+gui.xaero_display_hostile =顯示敵對的生物
+gui.xaero_box_entity_info =顯示您所指向的生物實體信息界面
+gui.xaero_entity_info_settings =實體信息設置
+gui.xaero_entity_info_stay =顯示實體信息
```
Translation back gives me something that makes no sense here. What does this mean?

the-eric-kwok replied:
This means "showing the info of entities".
I preferred "繼續顯示實體信息".

thexaero commented on zh_tw.lang:
```diff
@@ -0,0 +1,209 @@
+gui.xaero_auto_map_size =自動
+gui.xaero_reset_message =你確定你想重新設置設置嗎？
+gui.xaero_reset_message2 =此操作無法恢復。
+gui.xaero_entity_info_numbers =生命值
```
Does this mean displaying numbers instead of hearts?

the-eric-kwok replied:
Yes, the word "值" is value, so this should be numbers. But we called hearts in Minecraft as "生命值" too. So it's a bit confusing. To avoid that we can change it to "生命值（數值）".

thexaero commented on zh_tw.lang:
```diff
@@ -0,0 +1,209 @@
+gui.xaero_team_colours =團隊顏色
+gui.xaero_instant_waypoint =快速路徑點
+gui.xaero_entity_depth =顯示實體位置深度
+gui.xaero_show_full_amount =物品全數
```
Translates back as "all items" for me, which isn't correct.

the-eric-kwok replied:
The English version of this statement is also confusing. I translate it as "物品堆叠最大数量", which means the maximum amount of a stack of item.

thexaero commented on zh_tw.lang:
```diff
@@ -0,0 +1,209 @@
+gui.xaero_show_flowers =顯示畫
+gui.xaero_show_entity_model =顯示實體模型
+gui.xaero_waypoint_names =以上距離路徑點名稱
+gui.xaero_waypoints_distance =最大路徑點分佈
```
Does this talk about "distance"? Translation back seems incorrect.

the-eric-kwok replied:
It should be "路徑點最大渲染距離".
And the `gui.xaero_waypoint_names` should be "路徑點名稱顯示與距離之上" for me.

thexaero reviewed (commented):
Ok, I have gone through the file.
Overall, good job! I have left some comments about the lines that I'm either confused about or that seem incorrect to me. Please help me out with those!

aa89227-zz committed:
Update zh_tw.lang

remove space before the "="
[8]"初始" means the state of something, "初始化" is an action.
[11]corrected the translation. I think it was not very different.
[12]corrected the translation. 
[22]corrected the translation.
[23]corrected the translation.
[25]this doesn't affect my use of it. It would be strange to make some more correct translations.
[26]corrected the translation.
[37~45]most of them are corrected
[44]have removed?
[51]corrected the translation.
[149]corrected the translation.
[166]Does this refer to the official version?
[181]corrected the translation.
[183]Does this refer to a proportion or a range?
[187]I need more details of the usage.
[191]Not sure what "For" means
[198]corrected the translation.
[205]corrected the translation.
[206]corrected the translation.
[209]corrected the translation.

I just used Google Translation to translate "zh_cn.lang".
The simplified Chinese and traditional Chinese can be compared word by word, so "zh_cn.lang" also has a lot of problems.

thexaero commented on zh_tw.lang:
```diff
@@ -1,209 +1,209 @@
+gui.xaero_howto_drag=左鍵點擊並拖動來移動介面
+gui.xaero_howto_deselect=右键點擊取消選擇介面
+gui.xaero_howto_center=C鍵固定介面居中
+gui.xaero_different_centered=某些介面有著不同的居中方式
```
Still talks about "methods" when translated back for me.

thexaero commented on zh_tw.lang:
```diff
@@ -1,209 +1,209 @@
+gui.xaero_waypoints=路徑點
+gui.xaero_box_potion_effects=顯示藥水效果的介面
+gui.xaero_box_armour_status=顯示裝備狀態的介面
+gui.xaero_box_sprint=按下一個綁定的按鍵進行衝刺
```
Doesn't seem like the correct explanation. Can you translate these from the english file?

thexaero commented on zh_tw.lang:
```diff
@@ -1,209 +1,209 @@
+gui.xaero_fbo=測試版
+gui.xaero_potion_status_settings=藥水狀態設置
+gui.xaero_potion_names=藥水效果名稱
+gui.xaero_opacity=不透明
```
Is this "opacity"? Translates back as "opaque" for me.

thexaero commented on zh_tw.lang:
```diff
@@ -1,209 +1,209 @@
+gui.xaero_delete_set_message=你確定你想刪除當前設置嗎
+gui.xaero_delete_set_message2=此設置將會從目前的世界中刪除
+gui.xaero_toggle_waypoints=切換路徑點
+gui.xaero_dots_scale=實體點範圍
```
This is supposed to talk about entity dots. Translates back weird for me.

thexaero commented on zh_tw.lang:
```diff
@@ -1,209 +1,209 @@
+gui.xaero_entity_info=實體信息
+gui.xaero_toggle_slime=切換史萊姆區塊鏈
+gui.xaero_toggle_grid=切換區塊網格
+gui.xaero_compass_over_wp=指南針上的路徑點
```
It is used for displaying the compass letters on top of waypoints, instead of waypoints being on top.

thexaero commented on zh_tw.lang:
```diff
@@ -1,209 +1,209 @@
+gui.xaero_display_hostile=顯示敵對的生物
+gui.xaero_box_entity_info=顯示您所指向的生物實體信息介面
+gui.xaero_entity_info_settings=實體信息設置
+gui.xaero_entity_info_stay=保持實體信息
```
The "for" is there because it is followed by amount of time in seconds. Not sure if you need it here.

thexaero commented on zh_tw.lang:
```diff
@@ -1,209 +1,209 @@
+gui.xaero_auto_map_size=自動
+gui.xaero_reset_message=你確定你想重新設置設置嗎？
+gui.xaero_reset_message2=此操作無法恢復。
+gui.xaero_entity_info_numbers=生命值(數字)
```
This seems weird. The "numbers" is the most important word of this line.

thexaero commented on zh_tw.lang:
```diff
@@ -1,209 +1,209 @@
+gui.xaero_team_colours=團隊顏色
+gui.xaero_instant_waypoint=快速路徑點
+gui.xaero_entity_depth=顯示實體位置深度
+gui.xaero_show_full_amount=持有的物品數量
```
Doesn't seem correct. This shows the full amount of the item in your whole inventory.

thexaero commented on zh_tw.lang:
```diff
@@ -1,209 +1,209 @@
+gui.xaero_OK=確定
+gui.xaero_add_edit=添加/編輯
+gui.xaero_new_waypoint=新的路徑點
+gui.xaero_initial=初始化
```
But this shouldn't be an action. This is supposed to mean "initial" as in the first letter of a name.

thexaero reviewed (commented):
Thank you for the changes!
I have reviewed the zh_cn.lang file before including it in the mod, so it can't have such obvious mistakes. Don't trust Google Translate this much. It probably does have some mistakes though.
Added some new comments. I would prefer if you responded using the site forms though.

thexaero commented:
The author seems to have disappeared, so I'm closing this.
